src/Vehicle/EventHandler.h

```diff
@@ -56,6 +57,7 @@ class EventHandler : public QObject
     QTimer _timer;
     events::parser::Parser _parser;
     events::HealthAndArmingChecks _healthAndArmingChecks;
+    bool _healthAndArmingChecksValid{false};
```

Comment:
The variable value should be set on constructor initializer list, as the current class style.

Reply:
It is consistent, only members initialized from constructor arguments use the initializer list.